browser/resources/settings/brave_system_page/brave_performance_page.ts

```diff
@@ -11,7 +11,10 @@ import {getTemplate} from './brave_performance_page.html.js'
 import {pageVisibility} from '../page_visibility.js'
 import {loadTimeData} from '../i18n_setup.js'
 import '../performance_page/battery_page.js'
-import '../performance_page/performance_page.js'
+import '../performance_page/memory_page.js'
+import '../performance_page/tab_discard/exception_list.js';
+
+import type {ExceptionListElement} from '../performance_page/tab_discard/exception_list.js';
```

Comment:
need this here?

Reply:
Doesn't look like this is needed. Not sure why upstream has it in their file. Maybe just a leftover from previous refactoring. Removed. Thanks!